testsuite/MDAnalysisTests/coordinates/test_netcdf.py

```diff
@@ -795,6 +795,30 @@ def test_write_AtomGroup(self, universe, outfile, outtop):
                                       self.prec,
                                       err_msg="unitcells are not identical")
 
+    @pytest.mark.parametrize('var, expected', (
+        ('coordinates', 'angstrom'),
+        ('time', 'picosecond'),
+        ('cell_lengths', 'angstrom'),
+        ('cell_angles', 'degree'),
+        ('velocities', 'angstrom/picosecond'),
+    ))
+    def test_writer_units(self, universe, outfile, var, expected):
+        """Tests that the writer adheres to AMBER convention units
+        TODO: switch to input trajectory that also has force information.
+        """
+        trr = mda.Universe(GRO, TRR)
+
+        with mda.Writer(outfile, trr.trajectory.n_atoms, velocities=True,
+                        format='ncdf') as W:
+            for ts in trr.trajectory:
+                W.write_next_timestep(ts)
+
+        with netcdf.netcdf_file(outfile, mode='r') as ncdf:
+            unit = ncdf.variables[var].units.decode('utf-8')
+            assert_equal(unit, expected)
+
+        del trr
```

Comment:
The `del` here isn't necessary because (usually) once variables fall out of scope they automatically get del'd.

Reply:
Fixed in 85e4508. I have also moved the whole thing out of the `_NCDFWriterTest` class since the test was being run twice.